CMakeLists.txt

```diff
@@ -187,6 +204,14 @@ IF (WIN32)
 	SET(RTTR_LIBDIR "." CACHE INTERNAL "")
 	SET(RTTR_DOCDIR "." CACHE INTERNAL "")
 	SET(RTTR_DRIVERDIR "./driver" CACHE INTERNAL "")
+ELSEIF(APPLE)
+	# Apple wants an Apple Bundle
+	SET(RTTR_BINDIR "./s25client.app/Contents/MacOS/bin" CACHE INTERNAL "")
```

Comment:
I think `./s25client.app/Contents/MacOS` should rather be the Install prefix passed during cmake configure as not every apple user would want to create the `.app`-structure

Reply:
yes that nees to be adapted